compiler/src/dotty/tools/dotc/printing/PlainPrinter.scala

```diff
@@ -601,6 +601,8 @@ class PlainPrinter(_ctx: Context) extends Printer {
 
   def extendedLocationText(sym: Symbol): Text =
     if (!sym.exists) ""
+    else if isEmptyPrefix(sym.owner) then
+      " in the root package"
```

Comment:
shouldn't it be the empty package?
```suggestion
      " in the empty package"
```
> Top-level definitions outside a packaging are assumed to be injected into a special empty package. That package cannot be named and therefore cannot be imported. However, members of the empty package are visible to each other without qualification.

Reply:
Updated